_posts/blog/2019-06-20-Hello!I am Yini He.md

```diff
@@ -0,0 +1,76 @@
+
+
+## Where I come here  
+
+--As different as night and day（冰火两重天）
+
+I was born in Changsha city of China which famous for the spicy dishes and excessively distinct seasons. And then I moved to Haikou, which famous for the gentlest sea wind and spring-like seasons. As its title indicates, these two cities have extreme difference, just like a song plays from a gentle and poetic melody to passionate and fiery bursts, fully influencing the two sides of my personality. I am happy and passionate at work, at the same time, enjoy a totally peaceful and quiet life. 
+
+<figure>
+  <img src="/images/YiniHe/1coconut4.jpg">
+</figure>
+
+## What's my hobby
+
+-- The more National, the more International.（民族的就是世界的）
+
+I love travelling around the world. The coexistence of the classics with the prosperity of London attracts me incredibly.  The people there are gentle and full of artistic temperament.  Modern and beautiful Vancouver attracts me just as much.  The people there understand the beauty of cultural integration.  Da Nang beckons to me with it’s mysteriousness and simplicity. The people there are warm and kind. 
+
+<figure>
+  <img src="/images/YiniHe/2panda4.jpg">
+</figure>
+                  
+At the same time, I am interested in all kinds of traditional Chinese culture, with its food, traditional clothing, calligaraphy, tea and Guqin(an ancient instrument of China). With a history of 5000 years, China has preserved its ancient philosophy and beautiful civilization.  More often, I am fascinated by the merging of Chinese and western cultures. I used to play the ukelele wearing Qipao(a traditional dress of China).  I also tried cooking a Chinese meal with fish and chips. 
+
+<figure>
+  <img src="/images/YiniHe/3qipao3.jpg">
+</figure>
+
+# Life and work goals
+
+--Ideal comes from life（理想源于生活）
+
+My university major is psychology.  In my senior year, I decided to go to a special school and work as an intern teacher there.  The students at this school were suffering from mental disorder. This was the first time that I truly understood the importance of mental health at heart.  I love those children even when some of them say things that are illogical and incoherent, while still others just always sat in a corner. But there are those who have great artistic talents.  Some of them are attractive and have a friendly disposition, willing to be best friends with me. Ever since then, I’ve always thought about these teenagers and hope that they could have a better life with the help of medical and therapeutic advancements for mental health. 
```

Comment:
I've put these sentences on new lines and adjusted the language around "special school" to make it a little more inclusive. (It isn't generally accepted to say that people with mental health conditions or learning difficulties are "suffering"). BUT please check to make sure that I've not changed the meaning 😄 

```suggestion
My university major is psychology.
In my senior year, I decided to go to a special school and work as an intern teacher there.
The students at this school all had learning difficulties and some other mental health conditions.
This was the first time that I truly understood the importance of mental health at heart.
I love those children for all their unique differences.
There were some who had great artistic talents.
Others would sit quietly by themselves.
Some of the kids had a really friendly disposition and wanted to be best friends with me.
Ever since then, I’ve always thought about these teenagers and hope that they could have a better life with the help of social, medical and therapeutic advancements for mental health. 
```